website/docs/reference/appsmith-framework/widget-actions/download.md

```diff
@@ -3,48 +3,65 @@ sidebar_position: 6
 ---
 # Download
 
-This action is used to download any data as a file. We use [downloadjs](https://github.com/rndme/download) for this functionality
+With Appsmith, users have the ability to download a wide range of file formats. This action serves the purpose of downloading any data as a file. The feature is made possible by using  [downloadjs](https://github.com/rndme/download) library.
```

Comment:
Hey why did we change this? Was it being flagged by linter, else let us keep it as before

Reply:
Yes, there were a few lintr issues, so I rephrased the introduction.